translations/english/core-commands.ts

```diff
@@ -85,17 +85,13 @@ export const translations: Translations = {
 		"You can only add a Player to unrated battles.": "",
 		"Player must be set to \"p1\" or \"p2\", not \"${target}\".": "",
 		"User ${name} not found.": "",
-		"User ${name} must be in the battle room already.": "",
-		"This room already has a player in slot ${target}.": "",
```

Comment:
This translation is also still in use: https://github.com/smogon/pokemon-showdown/blob/5105cc46a377e3effbb3cf5afb9b0ee826b29593/server/chat-commands/core.ts#L968